owasp-top10-2017-apps/a9/cimentech/README.md

```diff
@@ -27,7 +27,7 @@ Then simply visit [localhost:80/][App] !
 
 ## Attack Narrative
 
-To understand how this vulnerability can be exploited, check this [other][Narrative] section!
+To understand how this vulnerability can be exploited, check [this section]!
```

Comment:
Suggestion: What do you think of `[this section](docs/ATTACK.md)`?

Reply:
Thanks @rafaveira3 ! But the link to `[this section]` is at the end of the file!